lib/tictactoe.rb

```diff
@@ -24,23 +26,31 @@ def show_board
     display.output(presenter.present(board))
   end
 
-  def play_turn
-    position = player.selection(board)
-    board.place_token(position)
+  def play_turn(player)
+    position = player.selection(validator)
```

Comment:
Can you elaborate on the decision to pass the validator into `selection`? Did you consider other alternatives? JC (just curious)

Reply:
I changed and now pass in board